plugins/actix-web/src/lib.rs

```diff
@@ -281,6 +293,42 @@ where
         call(self, spec)
     }
 
+    /// Exposes the previously built JSON specification with Swagger UI at the given path
+    ///
+    /// **NOTE:** you **MUST** call with_json_spec_at before calling this function
+    #[cfg(feature = "swagger-ui")]
+    pub fn with_swagger_ui_at(mut self, path: &str) -> Self {
+        let spec_path = self.spec_path.clone().expect(
+            "Specification not set, be sure to call `with_json_spec_at` before this function",
+        );
+
+        let path: String = path.into();
+        let regex_path = format!("{}/{{filename:.*}}", path);
```

Comment:
what is filename meant to be here?

Reply:
Hi @tiagolobocastro I have updated with a comment:

// Grab any file request from the documentation UI path and fetch it from SWAGGER_DIST
// E.g: js, html, svg and etc.

So when you have /swagger_ui/index.html or /swagger_ui/icons.svg, we can use the filename to fetch from the SWAGGER_DIST

Reply:
So what are we meant to pass to `with_swagger_ui_at` ? I would expect that we simply provide a path where we want the swagger_ui to be accessible, ex: /v0/swagger, and in that case it seems that we get redirected to the index.htlm?

So why would we need to specify a particular file?

Reply:
So, what happens is the following, `/v0/swagger` will be redirect to `/v0/swagger/index.html` with the url query parameter, the query will be populated with the path of the openapi/swagger definition file from `with_json_spec_at`. The redirect is also necessary to have a filename to fetch the file from SWAGGER_DIST and have a valid regex match.

Reply:
Oh right, got it :+1: